### ISSUE TITLE
chore: currently `libraries/knowledge-string.json` is checked in but should be elsewhere and .gitignore should ignore `libraries`

### ISSUE THREAD

**Issue by dalmaer** [polymath-ai/polymath-ai#23]
We don't want anyone to accidentally check in their libraries!

**Comment by m14t** [comment]
I'm willing to help here.

I see 3 library files checked in:
1. `core/client/libraries/knowledge-string.json`
    - It looks like this is used by the `core/client/tests/main.ts`
2. `kits/remix/libraries/knowledge-generators.json`
    - This is referenced in `kits/remix/app/config/generators.host.SAMPLE.json`:
3. `kits/remix/libraries/knowledge-string.json`
    - This is referenced in `kits/remix/README.md` and `kits/remix/app/config/local.host.SAMPLE.json`
    
This being the case, do you still want these files removed?  If so I could add some instructions on how they could be generated by a user.

**Comment by dglazkov** [comment]
Oh good question. I just recently checked in one more library file, at `core/db/tests/data/sample-library.json`. These little buggers are like gremlins!

My sense is that it is okay to have a test library file checked in, but it probably should live under `tests/data` in the right place. So for the first one, `core/client/tests/data/knowledge-string.json`.

The other two I am not so sure about. @dalmaer WDYT?

**Comment by dalmaer** [comment]
Yup, having test libraries mid totally great. I think we can get rid of the sample one in remix... it was for an easy "use this config and you can play with things".
